node/pkg/watchers/evm/ccq.go

```diff
@@ -732,22 +617,24 @@ type EthCallDataIntf interface {
 	CallDataList() []*query.EthCallData
 }
 
-func ccqBuildBatchFromCallData(req EthCallDataIntf, callBlockArg interface{}) ([]rpc.BatchElem, []EvmCallData) {
+// ccqBuildBatchFromCallData builds two slices. The first is the batch submitted to the RPC call. It contains one entry for each query plus one to query the block.
+// The second is the data associated with each request (but not the block request). The index into both is the index into the request call data.
+func ccqBuildBatchFromCallData(req EthCallDataIntf, callBlockArg interface{}) ([]rpc.BatchElem, EvmCallData) {
 	batch := []rpc.BatchElem{}
```

Comment:
there are two of these method names now - one on the watcher and one not. was the name change an accident?